### DISCUSSION TITLE
About backlog

### DISCUSSION THREAD

**Discussion by betulaydinturkk** [HYF-Class22/agile-development-group4-hyf-redesign#4] [Ideas]
**Must have**

- title
- responsive system
- introduction
- nav bar
- sign up

**Should have**

- photos
- contact
- map
- visual design

**Could have**

- support information parts
- partner companies

**Comment by betulaydinturkk** [comment]
Hi team, if you have another ideas you can add here.

**Reply by amirabayoumi** [reply]
well , normally that backlog we figure out after we create our user persona, but we can always edit it no problem
good job!
